common/changes/@microsoft/rush/remove-os-eol_2022-12-05-08-38.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "type": "none",
```

Comment:
This seems like a user-impacting change, for example if someone has Jest snapshots that diff Rush output.